package.json

```diff
@@ -29,7 +29,7 @@
     "@google-cloud/common": "^0.20.3",
     "bindings": "^1.2.1",
     "console-log-level": "^1.4.0",
-    "delay": "~3.0.0",
+    "delay": "^3.1.0",
```

Comment:
Shouldn't `@types/delay` also be removed?

Reply:
👍